pkgs/development/tools/misc/luarocks/luarocks-nix.nix

```diff
@@ -2,7 +2,7 @@
 
 luarocks.overrideAttrs (old: {
   pname = "luarocks-nix";
-  version = "unstable-2023-10-19";
+  version = "0-unstable-2023-10-19";
```

Comment:
The tags inherited from upstream `luarocks` only go up to `3.7.0`, but the master branch is clearly based on 3.9.2:

![image](https://github.com/NixOS/nixpkgs/assets/23431373/7f03b766-9d3b-4c8e-8068-0032f5669225)

…so there's no winning here, everything is wrong. :slightly_frowning_face: